tests/tekton-resources/tasks/addons/cw-metric.yaml

```diff
@@ -21,9 +21,17 @@ spec:
       description: "The unit of the metrics"
     - name: value
       description: "The value for the metric"
+    - name: is-obelisk
```

Comment:
I'm not in favor of this change.
This task should be generic. You should not change it here. You should pass `namespace=<foo>-obelisk` from outside of this task